ajax/libs/FlexSearch/package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "FlexSearch",
+  "description": "Next-Generation full text search library with zero dependencies.",
+  "author": "Thomas Wilkerling",
+  "license": "Apache-2.0",
+  "keywords": [
+    "search",
+    "fulltext",
+    "fastest",
+    "contextual",
+    "fuzzy",
+    "matching",
+    "scoring",
+    "indexing",
+    "fieldsearch"
+  ],
+  "filename": "",
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/nextapps-de/flexsearch.git"
+  },
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/nextapps-de/flexsearch.git",
+    "fileMap": [
+      {
+        "basePath": "dist",
+        "files": [
+          "flexsearch.min.js",
+          "flexsearch.compact.js",
+          "flexsearch.light.js",
+          "flexsearch.es5.js"
```

Comment:
Just to make sure, these are all working with browsers, right? Maybe try `"!(flexsearch.node.js)"` here to just ignore the one we don't need here?

Reply:
Thanks for the hint. All files are browser compatible. The flexsearch.node.js is also compatible but its just a special version for node.js (without web-worker parts).